README.md

```diff
@@ -30,8 +30,14 @@ To get started please navigate to [prerequisites](#prerequisites) and [getting s
 ## Prerequisites
 
 - Windows PowerShell version 5.1 / PowerShell 6+
+- NuGet
 - Pester
 
+To install package provider NuGet type in your PowerShell:
```

Comment:
We might want to adjust the wording here; folks who've already been installing modules from the gallery don't need this step, so maybe something along the lines of `If you've never installed PowerShell modules before, start by ensuring you have the NuGet package provider installed` etc.?

Reply:
Completely agree